test/testaux/validate_schema.py

```diff
@@ -0,0 +1,68 @@
+# Copyright © 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache License 2.0
+import pytest
+
+from libsigopt.aux.errors import InvalidTypeError, InvalidValueError
+from libsigopt.aux.validate_schema import validate
+
+
+class TestValidateSchema(object):
+  def test_validate_simple_array(self):
+    schema = {
+      "type": "array",
+      "items": {
+        "type": "object",
+      },
+    }
```

Comment:
can you add this as an example that passes: 
```
validate([{}], schema)
```
or maybe
```
validate([{}, {}], schema)
```
I think it's easier to understand the schema with a positive example and if you move them before the invalid examples that would be great